config

```diff
@@ -318,6 +318,7 @@ HTTP_LUA_SRCS=" \
             $ngx_addon_dir/src/ngx_http_lua_req_body.c \
             $ngx_addon_dir/src/ngx_http_lua_uri.c \
             $ngx_addon_dir/src/ngx_http_lua_args.c \
+            $ngx_addon_dir/src/ngx_http_lua_exten.c \
```

Comment:
I guess this should just go into the `ngx_http_lua_uri.c` compilation unit? No need to create a new CU here I think.